decentraland/unity-explorer
# Chore: camera reel behaviour critical fixes

## What does this PR change?

- handled overload exceptions
- add PlaceId when uploading sceenshot
- add cooldown to prevent screencapture spamming

## How to test the changes?

1. Launch the explorer
2. Verify you that you cannot make more than 500 images and and that uploading last (500th) image is handled properly - not present any issues.
3. Verify you cannot spam screenshoting

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
